src/main/java/com/exadel/discount/controller/UserController.java

```diff
@@ -0,0 +1,73 @@
+package com.exadel.discount.controller;
+
+import com.exadel.discount.dto.user.BaseUserDto;
+import com.exadel.discount.dto.user.UserDto;
+import com.exadel.discount.entity.User;
+import com.exadel.discount.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+@RestController
+@RequestMapping("/users")
+public class UserController {
+
+    private final UserService userService;
+
+    @Autowired
+    public UserController(UserService userService) {
+        this.userService = userService;
+    }
+
+    @PostMapping
+    public ResponseEntity<UserDto> addUser(@RequestBody final BaseUserDto baseUserDto) {
+        User user = userService.addUser(User.from(baseUserDto));
+        return new ResponseEntity<>(UserDto.from(user), HttpStatus.OK);
+    }
+
```

Comment:
I'm still sure that we can't add/delete/edit users. Our app dosen't have this task and we 100% will not provide registration, because product owner has said about it. Users - employees of our company, we can't register new employees.

Reply:
But it's additional feather. Not error or something bad. Leave it, please. I'll ask PO later.  Do not think it will be a problem.